.eslintrc

```diff
@@ -29,6 +29,9 @@
         "react/react-in-jsx-scope": 0,
         "wpcalypso/import-no-redux-combine-reducers": 0,
         "wpcalypso/jsx-classname-namespace": 0,
-        "wpcalypso/redux-no-bound-selectors": 1
+        "wpcalypso/redux-no-bound-selectors": 1,
+		"jsx-a11y/label-has-for": [ "error", {
+			"required": "id"
+		} ]
```

Comment:
Following Gutenberg's lead on this:
https://github.com/WordPress/gutenberg/blob/80ac62507567a7e8c2f02693bbb05edc1532407b/eslint/config.js#L44-L46